Bots/DotNet/Skills/Composer/ComposerSkillBotDotNet/ComposerSkillBotDotNet/scripts/DeploymentTemplates/function-template-with-preexisting-rg.json

```diff
@@ -0,0 +1,355 @@
+            "FUNCTIONS_EXTENSION_VERSION": "~3",
+            "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+            "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('microsoft.insights/components/', parameters('appInsightsName')), '2015-05-01').InstrumentationKey]",
+            "MicrosoftAppId": "[parameters('appId')]",
+            "MicrosoftAppPassword": "[parameters('appSecret')]"
+          }
+        }
+      ]
+    },
+    {
+      "comments": "CosmosDB for bot state.",
+      "type": "Microsoft.DocumentDB/databaseAccounts",
+      "kind": "GlobalDocumentDB",
+      "apiVersion": "2015-04-08",
+      "name": "[variables('cosmosDbAccountName')]",
+      "location": "[parameters('location')]",
+      "properties": {
+        "databaseAccountOfferType": "Standard",
+        "locations": [
+          {
+            "locationName": "[parameters('location')]",
+            "failoverPriority": 0
+          }
+        ]
+      },
+      "condition": "[parameters('useCosmosDb')]"
+    },
+    {
+      "type": "Microsoft.DocumentDB/databaseAccounts/sqlDatabases",
+      "apiVersion": "2020-03-01",
+      "name": "[concat(variables('cosmosDbAccountName'), '/botstate-db')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.DocumentDB/databaseAccounts', variables('cosmosDbAccountName'))]"
+      ],
+      "properties": {
+        "resource": {
+          "id": "botstate-db"
+        },
+        "options": {}
+      },
+      "condition": "[parameters('useCosmosDb')]"
+    },
+    {
+      "type": "Microsoft.DocumentDB/databaseAccounts/sqlDatabases/containers",
+      "apiVersion": "2020-03-01",
+      "name": "[concat(variables('cosmosDbAccountName'), '/botstate-db/botstate-container')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.DocumentDB/databaseAccounts/sqlDatabases', variables('cosmosDbAccountName'), 'botstate-db')]",
+        "[resourceId('Microsoft.DocumentDB/databaseAccounts', variables('cosmosDbAccountName'))]"
+      ],
+      "properties": {
+        "resource": {
+          "id": "botstate-container",
+          "indexingPolicy": {
+            "indexingMode": "consistent",
+            "automatic": true,
+            "includedPaths": [
+              {
+                "path": "/*"
+              }
+            ],
+            "excludedPaths": [
+              {
+                "path": "/\"_etag\"/?"
+              }
+            ]
+          },
+          "partitionKey": {
+            "paths": [
+              "/id"
+            ],
+            "kind": "Hash"
+          },
+          "conflictResolutionPolicy": {
+            "mode": "LastWriterWins",
+            "conflictResolutionPath": "/_ts"
+          }
+        },
+        "options": {}
+      },
+      "condition": "[parameters('useCosmosDb')]"
+    },
+    {
+      "apiVersion": "2017-12-01",
+      "type": "Microsoft.BotService/botServices",
+      "name": "[parameters('botId')]",
+      "location": "global",
+      "kind": "bot",

```

Comment:
The Microsoft.BotService/botServices resource kind "bot" is no longer supported. It needs to be updated to "azurebot" and its apiVersion to "2021-03-01".

Reply:
We'll remove these templates since, in this repo, they are centralized within the `build/templates` folder. 
Thanks!